forest/src/cli/wallet_cmd.rs

```diff
@@ -0,0 +1,208 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::str::FromStr;
+
+use address::Address;
+use forest_crypto::{
+    signature::{json::signature_type::SignatureTypeJson, SignatureType},
+    Signature,
+};
+use rpc_client::wallet_ops;
+use structopt::StructOpt;
+use wallet::json::KeyInfoJson;
+
+use super::{cli_error_and_die, handle_rpc_err};
+
+#[derive(Debug, StructOpt)]
+pub enum WalletCommands {
+    #[structopt(about = "Create a new wallet")]
+    New {
+        #[structopt(
+            default_value = "bls",
+            help = "The signature type to use. One of Secp256k1, or bls"
+        )]
+        signature_type: String,
+    },
+    #[structopt(about = "Get account balance")]
+    Balance {
+        #[structopt(about = "The address to of the account to check")]
```

Comment:
nit: "The address of the account to check"

Reply:
Good catch haha